packs/feats/ceremony-of-suns-gift.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "ancestry",
         "description": {
-            "value": "<p>Your connection to the sun has become an unbreakable bond. You don't become temporarily immune to the effects of your @UUID[Compendium.pf2e.ancestryfeatures.Item.Sunlight Healing], and instead of rolling to recover Hit Points, you gain the maximum number of HP possible (for example, 64 HP instead of 8d8).</p>\n<p>[[/r (ceil(@actor.level/2))*8]]{Sunlight Healing}</p>\n<p>[[/r (@actor.level)*8]]{Ceremony of Sunlight Healing}</p>"
+            "value": "<p>Your connection to the sun has become an unbreakable bond. You don't become temporarily immune to the effects of your @UUID[Compendium.pf2e.ancestryfeatures.Item.Sunlight Healing], and instead of rolling to recover Hit Points, you gain the maximum number of HP possible (for example, 64 HP instead of 8d8).</p>\n<p>@Damage[(max(1,ceil(@actor.level/2)))*8[healing]]{Sunlight Healing}</p>\n<p>@Damage[(max(1,@actor.level))*8[healing]]{Ceremony of Sunlight Healing}</p>"
```

Comment:
I don't believe it's possible to automate this yet, as DamageAlteration can only override dice, and not replace them with a static amount.